src/bin/bench-tps.rs

```diff
@@ -236,6 +354,18 @@ fn main() {
                 .takes_value(true)
                 .help("address to advertise to the network"),
         )
+        .arg(
```

Comment:
oh can you sneak in a `--sustained` here [1] as a part of this PR please.

[1] https://github.com/solana-labs/solana/blob/master/ci/testnet-deploy.sh#L165

Reply:
Sure